src/test/java/org/owasp/esapi/reference/EncoderTest.java

```diff
@@ -38,11 +38,13 @@
 import org.owasp.esapi.codecs.HTMLEntityCodec;
 import org.owasp.esapi.codecs.MySQLCodec;
 import org.owasp.esapi.codecs.OracleCodec;
+import org.owasp.esapi.codecs.JSONCodec;
```

Comment:
Surprise, surprise. Someone already alphabetized these imports (or perhaps we can blame an IDE).

Seriously, your email tagline should be:
_I troll anal people._